app/routes/user/credentials/index.tsx

```diff
@@ -61,6 +61,14 @@ export default function Index() {
           Add
         </Link>
       </div>
+      <div>
+        This page is for managing your generated User Credentials. The
+        Credentials are necessary for any programmatic access you want to set up
+        for receiving GCN Notification through the Kafka client. These
+        credentials are the same as those generated through the{' '}
+        <Link to="/quickstart">quickstart</Link>. For an example of how to use
+        these, see the <Link to="/docs/client">client docs</Link> page.
```

Comment:
1. [ ] We call them "client credentials," not "user credentials."
2. [ ] Rephrase to avoid using the phrase "This page is." (It's redundant, everything is a page.) Try having the first sentence in the imperative voice.
3. [ ] Replace "quickstart" with "Start Streaming GCN Notices." We never refer to it as the "quickstart" in the UI.
4. [ ] Avoid the phrase "see the <link> page" (also redundant).